SparkleDotNET/SUConstants.cs

```diff
@@ -27,6 +27,7 @@ public class SUConstants {
         public const string SU1stLaunchNotification = "SU1stLaunchNotification";
 
         //for bench stats
+        public const string Updated = "Updated";

```

Comment:
I'd prefer that these were named similarly. e.g.
DownloadComplete
or
UpdateComplete
UpdateCancelled
UpdateClicked

after that lgtm.